src/plugins/kibana/public/management/styles/main.less

```diff
@@ -234,9 +234,8 @@ kbn-management-indices {
   }
 }
 
-  .nav-buttons {
-    float:right;
-  }
+.nav-buttons {
```

Comment:
Are you sure you want to define this style here? Before it was confined to the `kbn-management-indices` directive. Now it's applied to the entire application. Also, where is this class/style being used?